aimmo-game/simulation/world_map.py

```diff
@@ -68,10 +68,23 @@ class WorldMap(object):
     """
 
     def __init__(self, grid):
-        self.grid = grid
+        self._grid = grid
+
+    @classmethod
+    def generate_empty_map(cls, height, width):
+        start_x = -width / 2 + 1
+        end_x = width / 2 + 1
+        start_y = -height / 2 + 1
+        end_y = height / 2 + 1
```

Comment:
Well it's a normal Python 2.7 fact. However, the behaviour of / has changed with Python 3, where it will return a float. So explicit flooring might be a good idea if we plan forward compatibility with Python3 (which we might like to use at some point...)